tests/std/tests/P0896R4_ranges_alg_uninitialized_copy/test.cpp

```diff
@@ -204,8 +204,8 @@ struct memcpy_test {
 
     static void call() {
         { // Validate matching ranges
-            int input[]  = {13, 55, 12345};
-            int output[] = {-1, -1, -1};
+            int input[] = {13, 55, 12345};
+            int output[3];

```

Comment:
My reasoning was that reading the indeterminate value will break the test, as during runtime we will most certainly break because the chance of comparing equal is slim. During constant evaluation we will get a hit in the face from the compiler

Reply:
(1) There is no "during constant evaluation" for these tests (not on this branch, anyway 😃).
(2) It's almost always a mistake to reason about the result of undefined behavior. What happens on this run may differ from what happens on the next, compiler flags may change the behavior, future compiler versions may do something completely different, etc. There's no reason to throw a grenade here and hope it does the right thing when we had a perfectly workable test a commit ago.